### PR TITLE
Task125 duplicate handling

### PR DESCRIPTION
**Pull request by jsight**

### CODE REVIEW

File: rules-java/api/src/main/java/org/jboss/windup/rules/apps/java/scan/provider/UnzipArchivesToOutputRuleProvider.java [hunk 1]
```diff
@@ -24,12 +38,48 @@
...
+            duplicateArchive.setOriginalArchive(originalArchive);
+            duplicateArchive.setSHA1Hash(originalArchive.getSHA1Hash());
+            duplicateArchive.setFilePath(originalArchive.getFilePath());
+            duplicateArchive.setArchiveName(originalArchive.getArchiveName());
+            duplicateArchive.setFileName(originalArchive.getFileName());
+            duplicateArchive.setParentArchive(originalParentArchive);
+            duplicateArchive.setParentFile(originalArchiveParentFile);
+
+            // Move the original archivemodel to the <shared-libs> application
+            duplicateArchive.setParentArchive(originalParentArchive);
```
Comment (d-s) [inline rules-java/api/src/main/java/org/jboss/windup/rules/apps/java/scan/provider/UnzipArchivesToOutputRuleProvider.java]: this looks like extra/leftover, you just set this four lines above, and the comment above it doesn't match

Comment (jsight) [inline rules-java/api/src/main/java/org/jboss/windup/rules/apps/java/scan/provider/UnzipArchivesToOutputRuleProvider.java]: Yep, that is wrong.

**Comment by jsight** [comment]
The dependents check failure is due to an API change. That can't really be fixed until this one is ready to be merged.

**Comment by jsight** [comment]
Completed in https://github.com/windup/windup/issues/936

**Pull request closed without merging**